surface/include/pcl/surface/ear_clipping.h

```diff
@@ -87,6 +87,14 @@ namespace pcl
         * \param[in] vertices the vertices representing the polygon 
         */
       float
+      area (const std::vector<index_t>& vertices);
+
+      /** \brief Compute the signed area of a polygon.
+        * \param[in] vertices the vertices representing the polygon
+        */
+      template <typename PointT, typename T = pcl::index_t, std::enable_if_t<!std::is_same<T, std::uint32_t>::value, pcl::index_t> = 0>
+      PCL_DEPRECATED(1, 13, "This method creates a useless copy of the vertices vector. Use  area method which accepts std::vector<index_t> instead")
```

Comment:
Deprecation needs to give 2 version delay. It should be either 15, I think (Or 14?)

Reply:
replace `std::vector<index_t> -> pcl::Indices`

Reply:
If this is going to be part of 12, then it should be 14 right?